lib/processor.js

```diff
@@ -79,7 +82,56 @@ function processInclude(type, tmpl, source) {
         : processJSInclude(tmpl, source);
 }
 
+/**
+ * Returns the height of the example block based on the line count
+ * @param {Number} lineCount - Count of lines in the source code
+ * @returns height - the value of the data-height property
+ */
+function getJSExampleHeightByLineCount(lineCount) {
+    if(lineCount <= MAX_LINE_COUNT_OF_SHORT_EXAMPLES) {
+        return 'shorter';
+    }
```

Comment:
```suggestion
    }
    
```

Reply:
Insert an empty line between two `if` statements

Reply:
With that said, is there a benefit to the two separate `if` statements as opposed to say:

```
if(lineCount <= MAX_LINE_COUNT_OF_SHORT_EXAMPLES) {
...
} else if (lineCount >= MIN_LINE_COUNT_OF_TALL_EXAMPLES) {
...
}
```

Reply:
Usually, I use separate `if` statements when I return from them. I prefer the `else-if` in case of assignments and a single return at the end of the function.
I am okay with both, if it fits better into our styleguide then I update it.